images/three-dots-white.svg

```diff
@@ -0,0 +1,32 @@
+<svg width="120" height="30" viewBox="0 0 120 30" xmlns="http://www.w3.org/2000/svg" fill="#FFFFFF">
```

Comment:
Where did you get this SVG from? I don't think you need it, there's a `sd-loader` CSS class available.

Reply:
> Where did you get this SVG from?

I got it from [here](https://github.com/superdesk/superdesk-client-core/blob/develop/images/three-dots.svg). I just change its color to white. 

![ezgif com-gif-maker](https://user-images.githubusercontent.com/10600741/96236954-520f4c00-0fba-11eb-84ed-d975ce7eb6c8.gif)
There are some issues because of which I didn't use the sd-loader class.

-  It shows the background color while hovering on it.
- On Hovering the width of the background color is overflowed.

So If I made any changes in the properties of sd-loader, It will reflect everywhere that's why I made these changes.